drivers/lis2dh12/lis2dh12.c

```diff
@@ -187,6 +194,72 @@ int lis2dh12_read(const lis2dh12_t *dev, int16_t *data)
     return LIS2DH12_OK;
 }
 
+int lis2dh12_set_int(const lis2dh12_t *dev, const lis2dh12_int_params_t *params, uint8_t int_line)
```

Comment:
I think it is better to do this not implicitly, because you may also want to change the interrupt behavior on run time. And therefor you dont need to reinitialize the bus.

Reply:
I'm okay with that, therefore I just said _could_ instead of _should_.